api/views.py

```diff
@@ -1,7 +1,32 @@
+from tracemalloc import start
 from django.shortcuts import render
 from django.http import HttpResponse
+from .utilities import create_discord_event
+import json
+
 
 # Create your views here.
 
 def create_event(request):
-    return HttpResponse("hello")
\ No newline at end of file
+    data = json.loads(request.body.decode('utf-8'))
+
+    name = data['name']
+    description = data['description']
+    start_time = data['start_time']
+    end_time = data['end_time']
+    location = data['location']
+    image = data['image']
+
```

Comment:
Before making the post url to the various event APIs, do you think we should do some validation to make sure that the name, description, start_time, end_time, location, and image are properly filled out, or do you think that's something that will primarily be handled on the front-end?

In my opinion, it's never a bad idea to do extra validation as something might go through accidentally on the front-end and it can help prevent internal server errors as well, espsecially as we're interfacing with external apis.

Maybe something that throws an error code and a message telling the user to properly format their request! (django response codes here: https://www.django-rest-framework.org/api-guide/status-codes/ )

Reply:
Will Handle in frontend